cluster/gce/windows/common.psm1

```diff
@@ -48,6 +48,25 @@ function Log-Output {
   }
 }
 
+# Dumps detailed information about the specified service to the console output.
+# $Delay can be set to a positive value to introduce some seconds of delay
+# before querying the service information, which may produce more consistent
+# results if this function is called immediately after changing a service's
+# configuration.
+function Write-VerboseServiceInfoToConsole {
+  param (
+    [parameter(Mandatory=$true)] [string]$Service,
+    [parameter(Mandatory=$false)] [int]$Delay = 0
+  )
+  if ($Delay -gt 0) {
```

Comment:
Add a TODO to remove the delay or rearrange the calls to this method so it's likely enough time has passed to get an accurate output of the service configuration.

Reply:
I don't understand. The delay is zero by default, but each callsite can set it independently "so it's likely enough time has passed to get an accurate output."

The callsites in this PR add a total of 3 seconds of delay. Are you saying that's too much?

Reply:
It's probably fine but a bit unfortunate.

Reply:
Yeah, I think it's ok for now. If/when we do an audit of node startup performance someday we can reconsider, but this will definitely not be the most significant delay.